proxy-attestation-server/Cargo.toml

```diff
@@ -23,7 +23,7 @@ sgx = ["sgx_types", "sgx_ucrypto", "sgx_urts", "psa-attestation/sgx", "transport
 # a feature to enable PSA native attestation.
 # Note: Final attestation is always PSA. This is just to enable platforms
 # that use PSA for their native attestation.
-psa = ["psa-attestation/tz"]
```

Comment:
The `psa-attestation/tz` feature instructs `psa-attestation`'s `build.rs` to use the cross-compiler shipped with the OP-TEE SDK to compile its native C code. I think this somehow worked because the TrustZone build was the only one where that code built by `build.rs` was actually linked.

My change to `psa-attestation`'s `build.rs` makes it use `$CC` as a default (which aligns with convention). Perhaps we should take that a step further and make `psa-attestation` not depend on platform feature flags, and instead always use `$CC` (which, again, aligns with convention).

We may have to set `$CC` in the other platform's build systems.